### PR TITLE
feat(EMI-1854): add source to partner offer

### PR DESCRIPTION
**Pull request by leamotta**
https://artsyproduct.atlassian.net/browse/EMI-1854

Support querying for https://github.com/artsy/gravity/pull/17669

### CODE REVIEW

File: src/schema/v2/partnerOfferToCollector.ts [hunk 1]
```diff
@@ -54,6 +54,9 @@ export const PartnerOfferToCollectorType = new GraphQLObjectType<
         )
       },
     },
+    source: {
+      type: GraphQLString,
```
Comment (fladson) [inline src/schema/v2/partnerOfferToCollector.ts:58]: Any reason for this not being enum?

Comment (leamotta) [reply src/schema/v2/partnerOfferToCollector.ts:58]: Good catch!

File: _schemaV2.graphql [hunk 2]
```diff
@@ -18522,6 +18524,11 @@ enum sort {
   DESC
 }
 
+enum SourceEnum {
```
Comment (fladson) [inline _schemaV2.graphql:18527]: 💅 Since this will be in the schema, I think we can prefix it with the model name, looks like it's the approach we use.

Comment (leamotta) [reply _schemaV2.graphql:18527]: Good point!

**Review by fladson — APPROVED** [review]

**Pull request merged**